server/etc/httpd/conf.d/pulp.conf

```diff
@@ -35,9 +35,11 @@ WSGISocketPrefix run/wsgi
 WSGIScriptAlias /pulp/api /usr/share/pulp/wsgi/webservices.wsgi
 WSGIImportScript /usr/share/pulp/wsgi/webservices.wsgi process-group=pulp application-group=pulp
 
-<Directory /usr/share/pulp/wsgi>
-    Require all granted
-</Directory>
+<IfVersion >= 2.4>
+    <Directory /usr/share/pulp/wsgi>
```

Comment:
No, I didn't look into the saneness (or lack there of) of any of the directives. I want to tackle that separately for issue 1257.